src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala

```diff
@@ -159,6 +161,16 @@ trait FireCloudApiService extends CookieAuthedApiService
     }
   }
 
+  // Return "cache-control: no-store" and "pragma: no-cache" headers,
+  // if those headers don't already exist on the response. This is included
+  // in `routeWrappers` below, so it affects all responses from Orch.
+  // Note that many Orch APIs are passthroughs, and if the underlying
+  // service (Rawls, Sam, etc) already returns these headers, Orch
+  // will not overwrite them.
+  private val noCacheNoStore: Directive0 = respondWithDefaultHeaders(
+    `Cache-Control`(`no-store`),
+    RawHeader("Pragma", `no-cache`.value))
```

Comment:
Why use `RawHeader` for `Pragma`, but not for `Cache-Control`?

Reply:
I couldn't find a pre-defined header definition for `Pragma` in `akka.http.scaladsl.model.headers.*`, so I had to resort to the `RawHeader` for it